hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/RaftNode.java

```diff
@@ -285,6 +286,13 @@ public void onError(PeerId peer, Status status) {
             }
         }
 
+        @Override
+        public void onBusy(PeerId peer, Status status) {
```

Comment:
seperate into 2 commits:
1. raft commit for: raft/* and raft-tools.sh
2. task commit for: TaskManager and rest-server.properties

Reply:
done